builder/imports/golang.py

```diff
@@ -0,0 +1,83 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+import os
+from pathlib import Path
+
+from builder.core.fetch import fetch_and_extract, mirror_package
+from builder.core.project import Import
+import builder.core.util as util
+
+URLs = {
+    'linux-armv6': 'https://go.dev/dl/go1.21.5.linux-armv6l.tar.gz',
```

Comment:
any specific reason for 1.25.5? 

if we just want latest versions, looks like we can them up here https://go.dev/dl/?mode=json